Meta/run.sh

```diff
@@ -183,6 +183,17 @@ else
     SERENITY_QEMU_DISPLAY_DEVICE="VGA,vgamem_mb=64 "
 fi
 
+# FIXME: Is there a better way of enabling NVME support?
+SERINITY_NVME_ENABLE=0
+if [ "$SERINITY_NVME_ENABLE" -eq 1 ]; then
+
```

Comment:
One nit: why is there an empty line here?

Reply:
Just bad eyesight! :P Fixed it! Thanks